packages/react/src/components/DatePicker/DatePicker.test.tsx

```diff
@@ -410,195 +399,136 @@ describe('DatePicker', () => {
       },
     );
   });
+});
 
-  describe('when Calendar properties are not specified', () => {
-    const datePicker = shallow(<DatePickerBase />);
-    datePicker
-      .find(TextField)
-      ?.props()
-      .onClick?.({} as any);
-    const calendarProps = datePicker.find(Calendar).props();
-
-    it('renders Calendar with isMonthPickerVisible as true by defaut', () => {
```

Comment:
what happened to these assertions ?

Reply:
![image](https://github.com/user-attachments/assets/1f5f7dd2-f8c3-4261-8985-165488443fe6)
still there ;)